### PR TITLE
feat(components): adapt design, use store for managing state

### PR DESCRIPTION
**Pull request by marymar**
adapt layout of components and use grid from vuetify, also arranged data
into nuxt vue store

BREAKING CHANGE: layout, data

### CODE REVIEW

File: components/content/LegalNotice.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="legal-notice">
+    <h4 class="legal-notice__title">{{ data.title }}</h4>
```
Comment (clehr) [inline components/content/LegalNotice.vue]: We are using BEM methodology here right?
I think in the long term for better readability we should switch to the utility first approach that's described here

https://adamwathan.me/css-utility-classes-and-separation-of-concerns/

but I think for now it is ok to go with this approach as our project is not that big yet.

File: components/content/Roles.vue [hunk 2]
```diff
@@ -1,79 +1,76 @@
...
-    />
-    <Organizer
-      :subpoints="[
-        'Assists in organizing meetups and talks',
-        'Actively helps to grow DevConnect',
-        'Helps with the social media presence'
-      ]"
-    />
-  </div>
+  <v-layout row wrap align-baseline class="roles">
```
[the remark this answers is not in the document]
Comment (marymar) [reply components/content/Roles.vue]: This line uses the layout component and classes to define the layout from vuetify.
That's the way how to apply CSS layout properties when vuetify is used.
It is similar to the bootstrap (Twitter) approach.

File: components/content/GetInvolved.vue [hunk 3]
```diff
@@ -1,31 +1,40 @@
 <template>
-  <div>
-    If you want to join us or even just want to have a look and say hello you
-    can find us under: <br /><br />
-    <div class="justify shorten center">
+  <v-layout row wrap align-baseline class="get-involved">
```
[the remark this answers is not in the document]
Comment (marymar) [reply components/content/GetInvolved.vue]: I think it is the way vuetify works.

File: components/content/Roles.vue [hunk 4]
```diff
@@ -1,79 +1,76 @@
...
+    <v-flex
+      v-for="role in data.types"
```
[the remark this answers is not in the document]
Comment (marymar) [reply components/content/Roles.vue]: I'll change it.

File: components/content/LegalNotice.vue [hunk 5]
```diff
@@ -0,0 +1,29 @@
+<template>
+  <div class="legal-notice">
```
[the remark this answers is not in the document]
Comment (marymar) [reply components/content/LegalNotice.vue]: In this case it is really specific to the LegalNotice component. It wouldn't be a reusable helper class.

**Pull request merged**